commit ad632fe2cc950e085a9f700dbef3664665739299
Author: Josh Wilsdon <jwilsdon@joyent.com>
Date:   2019-04-11T12:23:56-07:00 (6 months ago)
    
    OS-7736 would like vm uuid in debug messages when ignoring state changes in vmadmd
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/src/vm/sbin/vmadmd.js b/src/vm/sbin/vmadmd.js
index 4d293465..64f37c2e 100755
--- a/src/vm/sbin/vmadmd.js
+++ b/src/vm/sbin/vmadmd.js
@@ -903,8 +903,11 @@ function updateZoneStatus(ev)
         || (ev.oldstate === 'configured' && ev.newstate === 'incomplete')
         || (ev.oldstate === 'incomplete' && ev.newstate === 'installed')) {
         // just log it
-        log.debug({old: ev.oldstate, new: ev.newstate},
-            'ignoring state transitions before first boot');
+        log.debug({
+            old: ev.oldstate,
+            new: ev.newstate,
+            vm: ev.zonename
+        }, 'ignoring state transitions before first boot');
         return;
     }
 
